lib/request-form.js

```diff
@@ -119,7 +119,7 @@ var RequestView = Backbone.View.extend({
         view.timepicker.setDate(time)
       }
 
-      if (data.attributes.wheelchairAccessible) {
+      if (data.attributes.wheelchair) {
         view.$('#wheelchairAccessible').prop('checked', true)
```

Comment:
Good point.  I wavered on that because `wheelchair` seems less descriptive, but consistency is probably more important.